e2e-tests/demand-backup-eks-credentials/run

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+
+test_dir=$(realpath $(dirname $0))
+. ${test_dir}/../functions
+
+if [ $EKS -ne 1 ]; then
```

Comment:
@tplavcic  are you sure that we need to have separate test? maybe we can add just additional test case to existing one..

Reply:
@hors Well my initial idea was to change 1-2 tests to use this, just let them check if they are running in EKS environment and set/remove option accordingly. But for some reason I was under the impression that you and Natalia agreed on having a separate test for this so just went with it.

Reply:
I though about this. 
There some ideas:
1. To add checks in existing test: 
+ no extra tests
- we tangle our system (We already have a lots of if statements and it could be tricky to understand the test idea.)
- we change the test logic. Now we test situation - all clouds - all storages and it will be changed to aws cloud - aws storage. 
2. To add new test
- we have extra test similar to backup tests. 
+ we avoid extra if and save backup test logic. System looks simplier.
That is why I decided to add new tests. 
If it is not ok I can change demand-backup test, for example. Any problem with it! :) 
@hors @tplavcic

Reply:
@nmarukovich I think the idea with changing the old test was not to remove the logic for other clouds, but just to add another test case just for EKS.
But I agree it makes things simpler and cleaner with a separate test so let's proceed this way.